apis/druid/v1alpha1/druid_types.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 
 	appsv1 "k8s.io/api/apps/v1"
-	autoscalev2beta2 "k8s.io/api/autoscaling/v2beta2"
```

Comment:
Honestly I think it may be a bit early to fully remove support here?  This is only deprecated in 1.23 and removed in 1.26.  
For example AWS EKS only has 1.24 as its latest available version.
Maybe supporting both versions should be considered until a bit further down the line and then v2beta2 can be removed?

Reply:
We can revert, as of now the latest k8s version i tested it is with 1.25. Maintaining both versions might be some work.

Reply:
I have reverted to v2beta2